backend/app.py

```
@@ -163,18 +163,17 @@ def _get_gpu_status() -> str:
-                xpu_name = torch.xpu.get_device_name(0)
-            except Exception:
-                xpu_name = "Intel GPU"
-            return f"XPU ({xpu_name})"
-    except ImportError:
-        pass
+    # Intel XPU (Arc / Data Center) — native PyTorch 2.4+ support; IPEX optional
+    if hasattr(torch, "xpu") and torch.xpu.is_available():
+        try:
+            import intel_extension_for_pytorch  # noqa: F401 -- enhances XPU perf if available
+        except ImportError:
+            pass
```

Ruff (0.15.10)

[warning] Use `contextlib.suppress(ImportError)` instead of `try`-`except`-`pass`

(SIM105)